deliverables/README.md

```diff
@@ -11,17 +11,28 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+**What is an "interpreted" language?**<br>
+An interpreted language is where most implementations execute instructions directly without previously compiling a program into machine-language instructions. <br>
+**What is IRB?**<br>
+IRB stands for "interactive ruby" and allows the user to interactively execute Ruby expressions. <br>
+**What are Objects?**<br>
+Everything in Ruby is an object. ex: integers, characters, text, arrays, etc. <br>
+**What are Methods?**<br>
+Methods allow one object to interact with another. <br>
```

Comment:
Again, on the right track but an incomplete answer.  This is a characteristic of methods, i.e. a true statement about them, but it doesn't actually cover what a method *is*.